packages/admin-ui/src/filters/filter-disclosure.tsx

```diff
@@ -1,27 +1,29 @@
-import type { HTMLAttributes, ReactNode } from 'react'
+import type { ReactNode } from 'react'
 import React from 'react'
-import type { PickerStateReturn } from '../picker'
-import { PickerDisclosure } from '../picker'
+import { Button } from '../button'
+import { IconCaretUp } from '@vtex/phosphor-icons'
+import { MenuButton } from 'ariakit'
+
 import * as style from './filter-disclosure.style'
 
-export const FilterDisclosure = (
-  props: FilterDisclosureProps & HTMLAttributes<HTMLElement>
-) => {
-  const { state, children, ...labelProps } = props
+export const FilterDisclosure = (props: FilterDisclosureProps) => {
+  const { state, children } = props
 
   return (
-    <PickerDisclosure
-      state={state}
-      csx={style.baseline}
-      tabIndex={0}
-      {...labelProps}
-    >
+    <Button as={MenuButton as any} state={state} csx={style.baseline}>
       {children}
-    </PickerDisclosure>
+      <IconCaretUp
+        size="small"
+        csx={{
```

Comment:
You could encapsulate this **icons style** in the **filter disclosure style** file as a function and pass the `state.mounted` as a param to that function

Reply:
I agree with the suggestion, but I think you can create a `filter.style.ts` file and then you can create the following:

```ts
export const disclosure = style({...})

export const icons = (mounted: boolean) => style({...})

export const listItem = style({...})

export const list = style({...})

...
```

This way you centralize the styles in one place and you can import them into the files in which they are being used

Reply:
I think that's good lucas, but i'd rather make the full change later, as it will touch many files and already have another pr based on this one